src/comotion/dash.py

```diff
@@ -426,6 +455,263 @@ def stop(self):
+                file = parquet_buffer,
+                file_upload_response=file_upload_response
+            )
+            self.track_upload(df)
+            if commit_after_upload:
+                self.v2_commit_load(
+                    load=load,
+                    checksums=checksums
+                )
+
+        return load
+    
+    def v2_upload_csv(
+        self,
+        upload_files,
+        load: Load = None,
+        dash_orgname: str = None,
+        dash_table: str = None,
+        commit_after_upload: bool = False,
+        application_config: Optional[Dict[str,str]] = None,
+        checksums: Dict[str, Union[int, float, str]] = None,
+        modify_lambda: Callable = None,
+        load_type: str = 'APPEND_ONLY',
+        load_as_service_client_id: str = None,
+        partitions: Optional[List[str]] = None,
+        chunksize: int = 30000,  # Default chunksize to process CSV in chunks,
+        path_to_output_for_dryrun: str = None
+    ):
+        if not isinstance(upload_files, list):
+            upload_files = [upload_files]
+
+        if load is None:
+            print("Creating new load")
+            config = self.create_dashconfig(org_name=dash_orgname, application_config=application_config)
+            load = Load(config,table_name=dash_table, load_type=load_type,load_as_service_client_id=load_as_service_client_id,partitions=partitions)
+            print(f"Load ID: {load.load_id}")
+
+        for uploader in upload_files:
+            if isfile(uploader):
+                print(f"Uploading csv file: {uploader}")
+
+                # Reading CSV in chunks, converting each to Parquet, and uploading
+                
+                try:
+                    for chunk in pd.read_csv(uploader, chunksize=chunksize, dtype=object):
+                        self.v2_upload_df(
+                            df=chunk,
+                            load=load,
+                            modify_lambda=modify_lambda,
+                            path_to_output_for_dryrun=path_to_output_for_dryrun
+                        )
+                except Exception as e:
+                    raise ValueError(f"Issue with uploading chunk: {e}")
+                    
+                print("All chunks uploaded successfully")
+            elif isdir(uploader):
+                # Create list of files in directory to upload
+                upload_files = [join(uploader, upload_file) for upload_file in listdir(uploader) if isfile(join(uploader, upload_file))] # Create list of files in directory to upload
+                print(f"Uploading the following files: {upload_files}")
+                self.v2_upload_csv(
+                    upload_files = upload_files,
+                    load = load
+                )
+
+        if commit_after_upload:
+            self.v2_commit_load(
+                load=load,
+                checksums=checksums
+            )
+
+        return load
+    
+class Dash_v1_Easy_Upload():
```

Comment:
why do you need a whole class for v1 and v2.  PErhaps combine and call it UploadUtilities?

Reply:
also, unit tests

Reply:
upload utilities